.devcontainer/template.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "trustify-ui-tests",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "runArgs": [
+    "--privileged",
+    "--userns=keep-id",
+    "--network=host",
+    "--add-host=${HOSTNAME}:127.0.0.1",
+    "-e DISPLAY=:2"
+  ],
+  "remoteUser": "pwuser",
+  "features": {
+    "ghcr.io/devcontainers/features/desktop-lite:1": {
+      "VNC_RESOLUTION": "1920x1200x16"
+    }
+  },
+  "forwardPorts": [6080, 5901],
+  "portsAttributes": {
+    "6080": {
+      "label": "browser vnc"
+    },
+    "5901": {
+      "label": "real vnc"
+    }
+  },
+  "onCreateCommand": "npx playwright install-deps && npx playwright install",
+  "postCreateCommand": "npm ci",
+  "postStartCommand": "echo \"source /usr/share/bash-completion/completions/git\" >> ~/.bashrc",
```

Comment:
I am not sure why we needed the git completion on bash.

Reply:
I've removed it.